ioservice/io_foms.c

```diff
@@ -564,6 +564,7 @@
  */
 
 #define M0_OBJ_LAYOUT_ID(lid)  (lid & 0x0000ffffffffffff)
+#define M0_BYTECOUNT_USER_ID 8881212
```

Comment:
Move this to common place for io_foms and SNS foms.

Reply:
Done.